CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to irida-galaxy-importer will be documeted in this file.
 
+## 2.1.0
+* Added in support for importing IRIDA files that may be stored in the cloud.
```

Comment:
Could this be updated to state that it adds supports for IRIDA files that are not available locally (i.e. in the cloud)

Reply:
Sure! Updated in 62a41768af58e85053d84f4228e1e56642571e4a